pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -222,10 +222,10 @@ public void setUpNewTable(TableConfig tableConfig, IdealState idealState) {
 
     long currentTimeMs = getCurrentTimeMs();
     Map<String, Map<String, String>> instanceStatesMap = idealState.getRecord().getMapFields();
-    for (int partitionId = 0; partitionId < numPartitions; partitionId++) {
```

Comment:
nit: should we rename `numPartitions` to `numPartitionGroups` ?

Reply:
+1